sortinghat/core/recommendations/gender.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014-2021 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     Santiago Dueñas <sduenas@bitergia.com>
+#     Miguel Ángel Fernández <mafesan@bitergia.com>
+#     Eva Millán <evamillan@bitergia.com>
+#
+
+
+import logging
+import re
+
+import requests
+import urllib3.util
+
+from ..db import find_individual_by_uuid
+from ..errors import NotFoundError, InvalidValueError
+from django.conf import settings
```

Comment:
I would move this inside `_genderize` function and get the `GENDERIZE_API_KEY` only there. That will mean that the key will be read again if it's updated, so we won't need to restart the system again if the file changes. Also, the value of the key in memory will be (or should be) destroyed once the function is done.

Reply:
See my suggestion below. You can delete his line from above.